packages/react/package.json

```diff
@@ -22,10 +22,10 @@
   },
   "dependencies": {
     "@obosbbl/grunnmuren-icons-react": "workspace:^2.0.0-canary.1",
-    "@react-aria/utils": "^3.23.2",
+    "@react-aria/utils": "^3.24.0",
     "@types/node": "^20.11.19",
     "cva": "1.0.0-beta.1",
-    "react-aria-components": "^1.1.1"
+    "react-aria-components": "^1.2.0"
   },
   "peerDependencies": {
     "react": "^18"
```

Comment:
Vil vi ikke pinne disse avhengighetene?

Reply:
Synes ikke det, fordi dette er et lib. Bygger du custom checkboxes og bruker react aria må du installere avhengigheten selv, og hvis vi pinner er det fort gjort at du ender opp med to versjoner.